Lib/test/test_doctest/test_doctest2.txt

```diff
@@ -10,5 +10,5 @@ This test also has some (random) encoded (utf-8) unicode text:
 
   ï»¿ÐÐÐÐÐ
 
-This doesn't cause a problem in the tect surrounding the examples, but
+This doesn't cause a problem in the test surrounding the examples, but
```

Comment:
```suggestion
This doesn't cause a problem in the text surrounding the examples, but
```

I agree with your previous fix. C is closer to X than S on my keyboard. The examples are the doc-_tests_, embedded in the _text_.